package.json

```diff
@@ -46,6 +46,8 @@
     "btoa": "^1.2.1",
     "firebase": "^4.13.1",
     "node-fetch": "^1.7.2",
+    "request": "^2.86.0",
```

Comment:
Should these be in `devDependencies`? Here we'll pull them in at runtime as well.

Reply:
Yes, thanks for catching! Fixed :)

BTW for your context: Initial implementation involved encrypting service account private key using travis encrypted variables. This however didn't work our, as travis will not decrypt its variables for PRs from forked repos for security reasons (which makes sense), so we would not be able to see screenshots of failures in our PRs - which would be sad. After some deliberation I moved the code to a cloud function that accepts files an puts them on a dedicated team drive. Anyone can send it stuff, but there's nothing an attacker can gain, so I reckon it's good enough.

Reply:
Sounds good to me. Thanks!